lib-ee/emqx_license/src/emqx_license.erl

```diff
@@ -131,7 +132,10 @@ do_update({key, Content}, _Conf) when is_binary(Content); is_list(Content) ->
             #{<<"key">> => Content};
         {error, Reason} ->
             erlang:throw(Reason)
-    end.
+    end;
```

Comment:
I meant these lines:
https://github.com/emqx/emqx/pull/7023/files#diff-dd77d1b0c447a22667690f4781d0f1bf509f604556e5e82e53b09e0dad264fdeR132
https://github.com/emqx/emqx/pull/7023/files#diff-dd77d1b0c447a22667690f4781d0f1bf509f604556e5e82e53b09e0dad264fdeR121
in `do_update`.


Instead returning `#{<<"file">> => Filename}` and `#{<<"key">> => Content}` we should return something like:
```erlang
maps:without([<<"key">>], Conf#{<<"file">> => Filename})
```
and
```erlang
maps:without([<<"file">>], Conf#{<<"key">> => Content})
```
respectively, because `Conf`now keeps other parameters.